.gitignore

```diff
@@ -1,3 +1,13 @@
 .idea/
 *~
 .DS_Store
+build/
+_static/
+_templates/
+__pycache__
+.ipynb_checkpoints/
+*.ipynb
+*.pyc
+/dist/
+/*.egg-info
+/*.egg
```

Comment:
best practices thing: can clean up the `.gitignore` with comments on what we are ignoring and why. Example: https://github.com/github/gitignore/blob/master/Python.gitignore